### DISCUSSION TITLE
[@types/chrome] Add support for chrome.runtime.getContexts()

### DISCUSSION THREAD

**Discussion by thdoan** [DefinitelyTyped/DefinitelyTyped#66593] [Issues with a @types package]
Source:
https://developer.chrome.com/docs/extensions/reference/runtime/#method-getContexts

This also includes the new `ExtensionContext` type:
https://developer.chrome.com/docs/extensions/reference/runtime/#type-ExtensionContext

[a comment by an automated account is not shown]

**Comment by HereOrCode** [comment]
Any updates?

**Comment by ghost** [comment]
Added in #68195
